Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -125,38 +125,48 @@ public struct TableTopSceneView: View {
     
     /// Visualizes a new node added to the scene as an AR Plane.
     /// - Parameters:
+    ///   - renderer: The renderer for the scene.
     ///   - node: The node to be added to the scene.
     ///   - anchor: The anchor position of the node.
-    private func addPlane(with node: SCNNode, for anchor: ARAnchor) {
-        // Place content only for anchors found by plane detection.
+    private func addPlane(renderer: SCNSceneRenderer, node: SCNNode, for anchor: ARAnchor) {
```

Comment:
Since the method doesn't read like a sentence we shouldn't use the proposition for the final argument. I didn't notice this last review.

```suggestion
    private func addPlane(renderer: SCNSceneRenderer, node: SCNNode, anchor: ARAnchor) {
```